Inform where to add/change the value (#4429)
umbraco/UmbracoDocs: Umbraco-Cloud/Deployment/Content-Transfer/index.md

#### Umbraco version 9 and above

> For Deploy to handle Forms data as content, you'll need to ensure the `TransferFormsAsContent` setting is set to `true` in the `appsettings.json`file:

Reviewer: The sentence was being a bit wordy. How about something like this instead? 🤔 

```suggestion
For Deploy to handle Forms data as content, the `TransferFormsAsContent` setting found in the `appsettings.json` file needs to be `true`:
```

Author: Looks good @sofietoft 😄
